src/bond/assertions.cljc

```diff
@@ -0,0 +1,26 @@
+(ns bond.assertions
+  (:require [bond.james :as bond])
+  #?(:cljs (:require-macros [bond.james :as bond])))
+
+(defmacro called?
+  "An assertion to check if the spy was called at least 1 time."
+  [f & body]
+  `(bond/with-spy [~f]
+     (do ~@body)
+     (pos? (-> ~f bond/calls count))))
```

Comment:
I think I'd prefer a usage pattern like:
```clj
(bond/with-spy [my-fn]
  (is (some test expression))
  (is (called? my-fn)))
```

There's less magic and it composes decently with stubbing etc.

Reply:
I like that. How about this API, @gordonsyme?

```

Reply:
Yeah, that looks great